shared/naturalcrit/codeEditor/codeEditor.jsx

```diff
@@ -115,6 +131,9 @@ const CodeEditor = createClass({
 				'Shift-Cmd-Enter'  : this.newColumn,
 				'Ctrl-Enter'       : this.newPage,
 				'Cmd-Enter'        : this.newPage,
+				'Ctrl-F'           : 'findPersistent',
+				'Cmd-F'            : 'findPersistent',
+				'Shift-Enter'      : 'findPersistentPrevious',
```

Comment:
Shift Enter doesn't seem to do anything for me. What is the `findPersistentPrevious` search? Do we even need it?

Reply:
You'll want to open up the find dialog, then search for something. Enter will go to the next instance, shift enter should go to the previous instance.